_test/general.test.php

```diff
@@ -35,27 +94,328 @@ public function test_plugininfo() {
+     */
+    public function test_xml_response_is_valid_and_elements_are_set_accordingly_to_page_data_when_namespace_height_is_two()
+    {
+        $this->savePages(array('test123:test123'));
+
+        $xml = $this->getXML();
+
+        $ordernumbers = $xml->xpath('/findologic/items/item/allOrdernumbers/ordernumbers/ordernumber');
+        $ordernumber = $ordernumbers[0];
+
+        $attributes = $xml->xpath('/findologic/items/item/allAttributes/attributes/attribute');
+        $attributeKey = (string)$attributes[0]->key[0];
+        $attributeValue = (string)$attributes[0]->values[0]->value[0];
+
+        $expectedOrdernumber = 'test123:test123';
+        $expectedAttributeKey = 'cat';
+        $expectedAttributeValue = 'test123_test123';
+
+        $this->assertEquals($expectedAttributeKey, $attributeKey, 'Expected attribute key in XML should match the category value when namespace height is two "cat".');
+        $this->assertEquals($expectedAttributeValue, $attributeValue, 'Expected attribute value in XML should match the namespace formatted in a FINDOLOGIC proper format when namespace height is two "test123_test123".');
+        $this->assertEquals($expectedOrdernumber, $ordernumber, 'Expected ordernumber in XML should match the pages namespace when namespace height is two "test123:test123".');
+    }
+
+    /**
+     * Test to ensure that ordernumber and attribute categories in the XML are valid when DokuWiki has a namespace height of one.
+     */
+    public function test_xml_response_is_valid_and_elements_are_set_accordingly_to_page_data_when_namespace_height_is_one()
+    {
+        $this->savePages(array('test321'));
+
+        $xml = $this->getXML();
+
+        $ordernumbers = $xml->xpath('/findologic/items/item/allOrdernumbers/ordernumbers/ordernumber');
+        $ordernumber = $ordernumbers[0];
+
+        $attributes = $xml->xpath('/findologic/items/item/allAttributes/attributes/attribute');
+        $attributeKey = (string)$attributes[0]->key[0];
+        $attributeValue = (string)$attributes[0]->values[0]->value[0];
+
+        $expectedOrdernumber = 'test321';
+        $expectedAttributeKey = 'cat';
+        $expectedAttributeValue = 'test321';
+
+        $this->assertEquals($expectedAttributeKey, $attributeKey, 'Expected attribute key in XML should match the category value when namespace height is two "cat".');
+        $this->assertEquals($expectedAttributeValue, $attributeValue, 'Expected attribute value in XML should match the namespace formatted in a FINDOLOGIC proper format when namespace height is two "test123_test123".');
+        $this->assertEquals($expectedOrdernumber, $ordernumber, 'Expected ordernumber in XML should match the pages namespace when namespace height is two "test123:test123".');
+    }
+
+    /**
+     * Test to ensure that response is valid and parameters are correctly handled in the export call when
```

Comment:
Since the method name is pretty good I see no value in this comment.

Reply:
done